x/qgb/keeper/msg_server.go

```diff
@@ -25,186 +18,16 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 
 // ValsetConfirm handles MsgValsetConfirm.
 func (k msgServer) ValsetConfirm(
-	c context.Context,
-	msg *types.MsgValsetConfirm,
+	_ context.Context,
+	_ *types.MsgValsetConfirm,
 ) (*types.MsgValsetConfirmResponse, error) {
-	ctx := sdk.UnwrapSDKContext(c)
-
-	// Get valset by nonce
-	valset, found, err := k.GetValsetByNonce(ctx, msg.Nonce)
-	if err != nil {
-		return nil, err
-	}
-	if !found {
-		return nil, sdkerrors.Wrap(types.ErrAttestationNotFound, "valset attestation for nonce not found")
-	}
-
-	// Get orchestrator account from message
-	orchaddr, err := sdk.AccAddressFromBech32(msg.Orchestrator)
-	if err != nil {
-		return nil, sdkerrors.Wrap(types.ErrInvalid, "acc address invalid")
-	}
-
-	// Verify ethereum address match
-	if !common.IsHexAddress(msg.EthAddress) {
-		return nil, sdkerrors.Wrap(stakingtypes.ErrEthAddressNotHex, "ethereum address")
-	}
-	submittedEthAddress := common.HexToAddress(msg.EthAddress)
-
-	// Verify if signature is correct
-	bytesSignature, err := hex.DecodeString(msg.Signature)
-	if err != nil {
-		return nil, sdkerrors.Wrap(types.ErrInvalid, "signature decoding")
-	}
-	signBytes, err := valset.SignBytes(types.BridgeID)
-	if err != nil {
-		return nil, err
-	}
-	err = types.ValidateEthereumSignature(signBytes.Bytes(), bytesSignature, submittedEthAddress)
-	if err != nil {
-		return nil, sdkerrors.Wrap(
-			types.ErrInvalid,
-			fmt.Sprintf(
-				"signature verification failed expected sig by %s for valset nonce %d found %s",
-				submittedEthAddress.Hex(),
-				msg.Nonce,
-				msg.Signature,
-			),
-		)
-	}
-
-	// Check if the signature was already posted
-	_, found, err = k.GetValsetConfirm(ctx, msg.Nonce, orchaddr)
-	if err != nil {
-		return nil, sdkerrors.Wrap(err, "couldn't check for existing valset confirm")
-	}
-	if found {
-		return nil, sdkerrors.Wrap(types.ErrDuplicate, "signature duplicate")
-	}
-
-	// Persist signature
-	key, err := k.SetValsetConfirm(ctx, *msg)
-	if err != nil {
-		// Should we include more details in the error?
-		return nil, sdkerrors.Wrap(err, "couldn't set valset confirm")
-	}
-
-	ctx.EventManager().EmitEvent(
-		sdk.NewEvent(
-			sdk.EventTypeMessage,
-			sdk.NewAttribute(sdk.AttributeKeyModule, msg.Type()),
-			sdk.NewAttribute(types.AttributeKeyValsetConfirmKey, string(key)),
-		),
-	)
-
 	return &types.MsgValsetConfirmResponse{}, nil
 }
 
 // DataCommitmentConfirm handles MsgDataCommitmentConfirm.
 func (k msgServer) DataCommitmentConfirm(
-	c context.Context,
-	msg *types.MsgDataCommitmentConfirm,
+	_ context.Context,
+	_ *types.MsgDataCommitmentConfirm,
```

Comment:
[blocking]
perhaps we can leave a comment pointing to ADR005 here and in all of the other empty methods to describe why we're keeping empty ones.

Reply:
also, I think it would still be useful to emit an event that some confirm was submitted, we just don't store it in the state.

Reply:
Cool, will do, thanks

Reply:
Check  now please.
Also, I am leaving the handler as we might need it  down the line, not to have to hard fork or... Will remove it after implementing the orchestrator and relayer.